Makefile.inc1

```diff
@@ -2067,11 +2060,8 @@ package-pkg: .PHONY
 
 real-packages:	stage-packages create-packages sign-packages .PHONY
 
-real-update-packages: stage-packages .PHONY
-	${_+_}${MAKE} -C ${.CURDIR} PKG_VERSION=${PKG_VERSION} create-packages
-.if empty(PKG_VERSION_FROM_DIR)
-	@echo "==> Bootstrapping repository, not checking for new packages"
-.else
+real-update-packages: .PHONY
+.if defined(PKG_VERSION_FROM_DIR)
```

Comment:
The logic is now inverted for better flow.

Reply:
Does the inversion depend on your other changes or can it be merged standalone?